packages/core/src/components/Tabs/Tab.jsx

```diff
@@ -29,25 +29,35 @@ export class Tab extends React.PureComponent {
 
   render() {
     const classes = classnames('ds-c-tabs__item', this.props.className);
+    const sharedTabProps = {
+      className: classes,
+      id: this.props.id,
+      ref: tab => {
+        this.tab = tab;
+      }
+    };
 
-    return (
-      <a
-        aria-selected={String(this.props.selected)}
-        aria-disabled={String(this.props.disabled)}
-        aria-controls={this.props.panelId}
-        className={classes}
-        href={this.href}
-        id={this.props.id}
-        onClick={this.props.disabled ? undefined : this.handleClick}
-        onKeyDown={this.props.disabled ? undefined : this.handleKeyDown}
-        role="tab"
-        ref={tab => {
-          this.tab = tab;
-        }}
-      >
-        {this.props.children}
-      </a>
-    );
+    if (!this.props.disabled) {
+      return (
+        <a
+          role="tab"
```

Comment:
Could we move `role="tab"` to the `sharedTabProps`? You'll probably get yelled at by ESLint, but you can disable the error by placing the following above the `<a` 

```
// eslint-disable-next-line jsx-a11y/role-supports-aria-props
```